src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -603,7 +603,7 @@ private void WriteToConsole(ReadOnlySpan<char> value, bool transcribeResult, boo
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected strings to be the same, but they were different.
String lengths are both 14.
Strings differ at index 6.
Expected: 'Mac-1323.local'
But was:  'Mac-1351.local'
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 256
256:             $result[0].Source | Should -BeExactly $hostName
```